_posts/languages/nodejs/2000-01-01-common-errors.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Node.js Common Errors
+nav: Common Errors
+modified_at: 2020-06-12 00:00:00
+tags: nodejs
+index: 3
+---
+
+You may experience an error in your Node.js application that many customers
+faced when first deploying such application on Scalingo. Here is a list of the
+most common error messages.
+
+## devDependencies Also Contain the Build Dependencies
```

Comment:
```suggestion
## devDependencies Also Contain the Build Dependencies {#dev-dependencies}
```

not sure it will work

Reply:
I don't understand what you want to achieve here?

Reply:
I'd like to specify a HTML ID manually. I'd like it to overwrite the autogenerated HTML id. A shorter link would be easier to read and parse visually for our customers when this link will be added in emails and various other outputs (deployment logs?).

I suggest you accept the suggestion and we'll see what the ID will look like in the Review App.

Reply:
It works :+1: